<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatmapExample</class>
 <widget class="QWidget" name="HeatmapExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QCustomPlot" name="plot" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_cmap">
          <property name="text">
           <string>Color map:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_cmap">
          <item>
           <property name="text">
            <string>Grayscale </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Night</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Candy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Geography</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Thermal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spectrum</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hues</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_pixel">
          <property name="text">
           <string>Pixel resolution:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
          <item>
           <widget class="QLabel" name="label_001">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_pixels">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_200">
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_interp">
          <property name="text">
           <string>Interpolate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
